### PR TITLE
Create label templates

### PR DESCRIPTION
**Pull request by danidoni**
Depends on #16642 

This is how it looks like
image

### CODE REVIEW

File: src/api/app/assets/stylesheets/webui/validations.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.error-message {
+    color: var(--bs-danger-text-emphasis);
+}
```
Comment (hellcp-work) [inline src/api/app/assets/stylesheets/webui/validations.scss:3]: How about we use bootstrap with `invalid-feedback` class https://getbootstrap.com/docs/5.3/forms/validation/#server-side

Comment (danidoni) [reply src/api/app/assets/stylesheets/webui/validations.scss:3]: But that's what I use https://github.com/openSUSE/open-build-service/pull/16647/files#diff-a6452613bd0bcdcd8c3746c569bad8a787497c8b88eec3e1b4d4037d58325180R4

Comment (danidoni) [reply src/api/app/assets/stylesheets/webui/validations.scss:3]: Ah, for the message also... Let me see...

Comment (danidoni) [reply src/api/app/assets/stylesheets/webui/validations.scss:3]: Done

Comment (hellcp-work) [reply src/api/app/assets/stylesheets/webui/validations.scss:3]: So we don't need this css?

Comment (danidoni) [reply src/api/app/assets/stylesheets/webui/validations.scss]: We don't, indeed

**Review by rubhanazeem — APPROVED** [review]

**Pull request merged**